test/unique_ptr_test.cpp

```diff
@@ -1,81 +1,99 @@
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
 #include <gtest/gtest.h>
 
-TEST(PluginlibUniquePtrTest, unknownPlugin)
-{
+#include <pluginlib/class_loader.h>
+
+#include "test_base.h"  // NOLINT
+
+TEST(PluginlibUniquePtrTest, unknownPlugin) {
   pluginlib::ClassLoader<test_base::Fubar> test_loader("pluginlib", "test_base::Fubar");
-  ASSERT_THROW(test_loader.createUniqueInstance("pluginlib/foobar"), pluginlib::LibraryLoadException);
+  ASSERT_THROW(test_loader.createUniqueInstance("pluginlib/foobar"),
+    pluginlib::LibraryLoadException);
 }
 
 
-TEST(PluginlibUniquePtrTest, misspelledPlugin)
-{
+TEST(PluginlibUniquePtrTest, misspelledPlugin) {
   pluginlib::ClassLoader<test_base::Fubar> bad_test_loader("pluginlib", "test_base::Fuba");
-  ASSERT_THROW(bad_test_loader.createUniqueInstance("pluginlib/foo"), pluginlib::LibraryLoadException);
+  ASSERT_THROW(bad_test_loader.createUniqueInstance(
+      "pluginlib/foo"), pluginlib::LibraryLoadException);
 }
 
-TEST(PluginlibTest, brokenPlugin)
-{
+TEST(PluginlibTest, brokenPlugin) {
   pluginlib::ClassLoader<test_base::Fubar> test_loader("pluginlib", "test_base::Fubar");
   ASSERT_THROW(test_loader.createUniqueInstance("pluginlib/none"), pluginlib::PluginlibException);
 }
 
-TEST(PluginlibUniquePtrTest, workingPlugin)
-{
+TEST(PluginlibUniquePtrTest, workingPlugin) {
   pluginlib::ClassLoader<test_base::Fubar> test_loader("pluginlib", "test_base::Fubar");
 
-  try
-  {
+  try {
     pluginlib::UniquePtr<test_base::Fubar> foo = test_loader.createUniqueInstance("pluginlib/foo");
     foo->initialize(10.0);
-    EXPECT_EQ(foo->result(),100.0);
-  }
-  catch(pluginlib::PluginlibException& ex)
-  {
+    EXPECT_EQ(foo->result(), 100.0);
+  } catch (pluginlib::PluginlibException & ex) {
     FAIL() << "Throwing exception: " << ex.what();
     return;
-  }
-  catch(...)
-  {
+  } catch (...) {
     FAIL() << "Uncaught exception";
   }
 }
 
-TEST(PluginlibUniquePtrTest, createUniqueInstanceAndUnloadLibrary)
-{
-  ROS_INFO( "Making the ClassLoader..." );
+TEST(PluginlibUniquePtrTest, createUniqueInstanceAndUnloadLibrary) {
+  ROS_INFO("Making the ClassLoader...");
   pluginlib::ClassLoader<test_base::Fubar> pl("pluginlib", "test_base::Fubar");
 
-  ROS_INFO( "Instantiating plugin..." );
+  ROS_INFO("Instantiating plugin...");
   {
-   pluginlib::UniquePtr<test_base::Fubar> inst = pl.createUniqueInstance("pluginlib/foo");
+    pluginlib::UniquePtr<test_base::Fubar> inst = pl.createUniqueInstance("pluginlib/foo");
   }
 
-  ROS_INFO( "Checking if plugin is loaded with isClassLoaded..." );
-  if( pl.isClassLoaded( "pluginlib/foo" ) )
-    ROS_INFO( "Class is loaded" );
-  else
-  {
-    FAIL() <<  "Library containing class should be loaded but isn't.";
+  ROS_INFO("Checking if plugin is loaded with isClassLoaded...");
+  if (pl.isClassLoaded("pluginlib/foo") ) {
```

Comment:
Nit: extra whitespace before closing parenthesis (same below)

Reply:
Yeah, I've noticed that uncrustify doesn't catch this always, I briefly investigated it but couldn't find a good setting to fix it.